tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/SqlInjection/EFCoreBaseTests.cs

```diff
@@ -11,6 +11,9 @@ namespace Samples.InstrumentedTests.Iast.Vulnerabilities.SqlInjection;
 
 public abstract class EFCoreBaseTests: InstrumentationTestsBase, IDisposable
 {
+    [CollectionDefinition("Non-Parallel Collection", DisableParallelization = true)]
```

Comment:
nit naming: basically the only reason we put tests in collection definitions is to _not_ run them in parallel, and in fact, we _always_ make sure to run tests in _any_ collection not-in parallel. Therefore we generally favor more declarative names. (We also tend to use `nameof()` to avoid typos, because it's impossible to know if you have a typo, but that doesn't necessarily make sense in this case)
 
```suggestion
    [CollectionDefinition("IastEfCoreSqliteTests", DisableParallelization = true)]
```